src/accountingservice/main.go

```diff
@@ -27,24 +26,14 @@ import (
 	"github.com/open-telemetry/opentelemetry-demo/src/accountingservice/kafka"
 )
 
-var log *logrus.Logger
 var resource *sdkresource.Resource
 var initResourcesOnce sync.Once
 
-func init() {
-	log = logrus.New()
-	log.Level = logrus.DebugLevel
-	log.Formatter = &logrus.JSONFormatter{
-		FieldMap: logrus.FieldMap{
-			logrus.FieldKeyTime:  "timestamp",
-			logrus.FieldKeyLevel: "severity",
-			logrus.FieldKeyMsg:   "message",
-		},
-		TimestampFormat: time.RFC3339Nano,
-	}
-	log.Out = os.Stdout
+func initLogger() *slog.Logger {
+	logger := slog.New(slog.NewJSONHandler(os.Stderr, nil)).With("service", "accounting")
+	slog.SetDefault(logger)
```

Comment:
Why are you setting the default logger. You never use it afterwards.

Consider removing this line or not returning the logger in this function and using `slog.LogAttrs` instead of `logger.LogAttrs`.

Reply:
Okay I will do the changes as requested
Thank you for the reviews 😊